Use portable lib for finetuning demo

dpalmasan opened the pull request:
Summary: Update the example of LLM fine-tuning on ET using portable_lib and XNNPack backend

Differential Revision: D69954671

swolchok commented on examples/llm_pte_finetuning/model_loading_lib.py:
```diff
@@ -72,16 +73,70 @@ def export_model_lora_training(
+                node.op == "call_function"
+                and node.target == torch.ops.aten.empty_permuted.out
+            ):
+                print("find empty_permute: ", node)
```
nit: I think the past tense "found" is more typical for log output than the present tense "find"

dpalmasan replied:
Makes sense. Will update it, thanks!

JacobSzwejbka reviewed (approved):
Looks good to me. cc @mergennachin who had been thinking about using a different repo for examples.

dpalmasan committed:
Use portable lib for finetuning demo (#8613)

Summary:

Update the example of LLM fine-tuning on ET using portable_lib and XNNPack backend

Reviewed By: JacobSzwejbka

Differential Revision: D69954671
